arwen/host/arwen.go

```diff
@@ -311,69 +306,55 @@ func (host *vmHost) RunSmartContractCreate(input *vmcommon.ContractCreateInput)
 	host.mutExecution.RLock()
 	defer host.mutExecution.RUnlock()
 
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*2)
```

Comment:
maybe define as a const the time.Second*2 and use it also on 334

Reply:
done